assets/css/navbar.css

```diff
@@ -0,0 +1,20 @@
+#header {
+  z-index: 1;
+  background-color: var(--white);
+  width: 100%;
+  position: fixed;
+  left: 0;
+  top: 0;
+  border-bottom: 1px solid var(--neutral-200);
+}
+
+.navbar {
```

Comment:
Thanks for adding this, I might change it tho, because I'm working on this component currently

Reply:
Okay sure.